rust/noosphere-ns/src/bin/bootstrap_nns/main.rs

```diff
@@ -0,0 +1,38 @@
+#![cfg(not(target_arch = "wasm32"))]
+
+mod cli;
+mod runner;
+mod utils;
+use anyhow::{anyhow, Result};
+use clap::Parser;
+use cli::{CLICommand, CLI};
+use noosphere::key::InsecureKeyStorage;
+use runner::{Runner, RunnerConfig};
+use tokio;
+
+#[tokio::main]
+async fn main() -> Result<()> {
+    let key_storage = InsecureKeyStorage::new(&utils::get_keys_dir()?)?;
+
+    let mut runner = match CLI::parse().command {
+        command @ CLICommand::Run { .. } => {
+            Runner::from(RunnerConfig::try_from_command(&key_storage, command).await?)
+        }
+        _ => {
+            return Err(anyhow!("Unimplemented."));
+        }
+    };
+
+    // Allow aborting (ctrl+c) during the initial run,
+    // and after (when we want to wait exclusively for ctrl+c signal)
+    let mut aborted = false;
+    tokio::select! {
+        _ = tokio::signal::ctrl_c() => { aborted = true; },
```

Comment:
Maybe there's a better way here, e.g. have `runner.run()` sleep the thread after setting things up. Would like to avoid signaling/messaging INTO runner to abort, unless there's a need for it